pom.xml

```diff
@@ -424,6 +448,10 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.22.0</version>
+          <!--I get weird test failures without this-->
+          <configuration>
+            <useSystemClassLoader>false</useSystemClassLoader>
```

Comment:
We need to get to the bottom of this change

Reply:
Yeah. Should I split it out into a separate issue?

Reply:
Let's wait with this, I will try to run to debug it before merging

Reply:
Removed